projects/ngx-raptorize/package.json

```diff
@@ -13,7 +13,6 @@
     "tslib": "^2.0.0"
   },
   "peerDependencies": {
-    "@angular/common": "^9.0.7",
-    "@angular/core": "^9.0.7"
+    "@angular/core": ">=9.0.0"
```

Comment:
I didn't found any import of `@angular/common` 🤷🏻

Reply:
Angular generate added these lines.
Thanks I would know now :)